MediaRecorder.bs

```diff
@@ -286,11 +276,35 @@ interface MediaRecorder : EventTarget {
     <li>Set |recorder|'s {{state}} to {{recording}}, and run the following steps
     in parallel:
     <ol>
-      <li>Start recording all tracks in |tracks| using the {{MediaRecorder}}'s
-      current configuration, and gather the data into a {{Blob}} <var>blob</var>
-      and queue a task, using the DOM manipulation task source, to
-      <a>fire an event</a> named <a>start</a> at <var>recorder</var>, then return
-      <code>undefined</code>.</li>
+      <li>Let the User Agent specify the |recorder|'s configuration for
+      recording all the tracks in |tracks|, given its current constraints. If
+      track data is available, the User Agent MAY take the content into account
+      when making this decision as long as this does not violate how the
+      configuration has previously been constrained (see earlier steps).</li>
```

Comment:
I just realized: If we want re-encoding to be allowed to be avoided "_only when mimeType is unset by JS_" as we decided at TPAC, I might need to add a sentence about not being allowed to avoid re-encoding is [[ConstrainedMimeType]] is non-null.

As-is I think the steps are correct for always deferring mimeType which I think we want, but the note makes it sound like we are allowed to avoid re-encoding as long as we don't violate the constraints of [[ConstrainedMimeType]], but if we want explicitly asking for VP8 to cause re-encoding then we should clarify that.

But @Pehrsons please take a look the way it is written right now anyway to make sure we're on the right track.